packages/oauth/oauth_server.js

```diff
@@ -342,7 +342,10 @@ var renderEndOfLoginResponse = function (options) {
     throw new Error('invalid loginStyle: ' + options.loginStyle);
   }
 
-  var result = template.replace(/##CONFIG##/, JSON.stringify(config));
+  var result = template.replace(/##CONFIG##/, JSON.stringify(config))
+    .replace(
+      /##ROOT_URL_PATH_PREFIX##/, __meteor_runtime_config__.ROOT_URL_PATH_PREFIX
```

Comment:
Can these `RegExp`s appear more than once? If so, you should use `/##ROOT_URL_PATH_PREFIX##/g` to match multiple times. Otherwise only the first occurrence will be replaced. If you don't think anything needs to be changed, just comment to let me know you've seen this question.